polkadot-launch/launch-rococo-local-with-encointer-and-sybil-dummy.json

```diff
@@ -24,7 +24,6 @@
 		{
 			"bin": "../target/release/encointer-collator",
 			"chain": "encointer-rococo-local",
-			"id": "1862",
 			"balance": "1000000000000000000000",
```

Comment:
`--parachain-id` is no longer supported by the collators. We can only use the ones from the chain-specs, but we have been doing that lately anyhow.

They removed it because it was error-prone.